src/tests/Microsoft.Diagnostics.NETCore.Client/KeyedHandleableCollectionTests.cs

```diff
@@ -0,0 +1,443 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Diagnostics.NETCore.Client
+{
+    public class KeyedHandleableCollectionTests
+    {
+        private static readonly TimeSpan DefaultHandleTimeout = TimeSpan.FromMilliseconds(50);
```

Comment:
Same concerns as in the other handleable collectiontest: this timeout may insufficient for CI.

Reply:
I'll bump all of the test timeouts significantly to allow for slower or more constrained machines.